olp-cpp-sdk-dataservice-read/src/repositories/PrefetchTilesRepository.cpp

```diff
@@ -32,7 +32,10 @@ namespace dataservice {
 namespace read {
 namespace repository {
 
-#define LOGTAG "PREFETCH_TILES_REPOSITORY"
+namespace {
+constexpr char kPTRLogtag[] = "PREFETCH_TILES_REPOSITORY";
```

Comment:
Replace tag name with CamelCase in order to maintain consistency with other LogTags formats.
```suggestion
constexpr char kPTRLogtag[] = "PrefetchTilesRepository";
```